tests/ClosestCity/ClosestCity.cpp

```diff
@@ -15,119 +15,189 @@
+        << "We should have one " << CityNames[2] << " but we found"
+        << findCity(vec, CityNames[2]);
+    ASSERT_TRUE(findCity(vec, CityNames[4]))
+        << "We should have one " << CityNames[4] << " but we found"
+        << findCity(vec, CityNames[4]);
+    ASSERT_FALSE(findCity(vec, CityNames[3]))
+        << "We should have none " << CityNames[3]
+        << " but it is still in the vector so the function "
+           "removeCitiesInVectorThatAreSmallerThanMe is working wrong";
+
     vec.clear();
     vec = DatabaseReader::readCitiesFromFile(path);
-    ClosestCity::removeCitiesInVectorThatAreSmallerThanMe(vec,CityNames[3]);
-   ASSERT_EQ(10, vec.size()) << "We should erase none city as we have nocity smaller than Katowice in vector but we have: " << vec.size() << " of them";
-    ASSERT_TRUE(findCity(vec,CityNames[0])) << "We should have one " << CityNames[0] << " but we found" << findCity(vec,CityNames[0]);
-    ASSERT_TRUE(findCity(vec,CityNames[1])) << "We should have one " << CityNames[1] << " but we found" << findCity(vec,CityNames[1]);
-    ASSERT_TRUE(findCity(vec,CityNames[2])) << "We should have one " << CityNames[2] << " but we found" << findCity(vec,CityNames[2]);
-    ASSERT_TRUE(findCity(vec,CityNames[3])) << "We should have one " << CityNames[3] << " but we found" << findCity(vec,CityNames[3]);
-    ASSERT_TRUE(findCity(vec,CityNames[4])) << "We should have one " << CityNames[4] << " but we found" << findCity(vec,CityNames[4]);
-    
+    ClosestCity::removeCitiesInVectorThatAreSmallerThanMe(vec, CityNames[3]);
+    ASSERT_EQ(10, vec.size()) << "We should erase none city as we have nocity "
+                                 "smaller than Katowice in vector but we have: "
+                              << vec.size() << " of them";
+    ASSERT_TRUE(findCity(vec, CityNames[0]))
+        << "We should have one " << CityNames[0] << " but we found"
+        << findCity(vec, CityNames[0]);
+    ASSERT_TRUE(findCity(vec, CityNames[1]))
+        << "We should have one " << CityNames[1] << " but we found"
+        << findCity(vec, CityNames[1]);
+    ASSERT_TRUE(findCity(vec, CityNames[2]))
+        << "We should have one " << CityNames[2] << " but we found"
+        << findCity(vec, CityNames[2]);
+    ASSERT_TRUE(findCity(vec, CityNames[3]))
+        << "We should have one " << CityNames[3] << " but we found"
+        << findCity(vec, CityNames[3]);
+    ASSERT_TRUE(findCity(vec, CityNames[4]))
+        << "We should have one " << CityNames[4] << " but we found"
+        << findCity(vec, CityNames[4]);
+
     // testing with flags and messages
     vec.clear();
     vec = DatabaseReader::readCitiesFromFile(path);
-    
-    std::string NextCity = ClosestCity::getTheClosestCityThatIsBiggerThanMe(vec,CityNames[0]);
 
-   ASSERT_EQ("", NextCity) << "As Warsaw is the biggest city we expect the function to return us empty string "" but we got: " << NextCity;
-    
+    std::string NextCity =
```

Comment:
lowerCase, please

Reply:
Well, it looks better like this in my opinion unless there is some overall clean code rule. Therefor I will leave it now and ask you for why you request this?

Reply:
@kermit10000000 I thought variable names should be lowercase. It is general rule, isn't it?
But merge anyway, it could be changed someday in the future.

Reply:
Well, maybe yeah. I do not remember honestly There is no problem for me of course to go for it but I believe that rule should be stated somewhere if we want each contributor to follow it.